src/repl.ml

```diff
@@ -174,15 +226,14 @@ module Command = struct
         match term with
         | Error err -> show_message `Error "Could not start the REPL: %s" err
         | Ok term ->
-          let code = get_code textEditor in
-          if String.length code > 0 then
-            let code = prepare_code code in
-            Terminal_sandbox.send term code
+          let uri = get_uri textEditor in
+          let use = "#use \"" ^ uri ^ "\";;" in
```

Comment:
`get_uri` will return paths with backslashes like `c:\a\b\c` when it uses `Uri.fsPath` on Windows. [Quoted string literals](https://v2.ocaml.org/manual/lex.html#sss:stringliterals) are more appropriate in this situation.

```suggestion
          let use = "#use {|" ^ uri ^ "|};;" in
```